src/Dropdown.js

```diff
@@ -11,7 +11,7 @@ const DropdownBase = ({title, children, className, ...rest}) => {
   return (
     <Details overlay className={className} {...rest}>
       <>
-        <Button as="summary" color="gray.7" aria-haspopup="true" {...rest}>
```

Comment:
The `color` here is already set in the `Button` component and isn't needed, so I removed it